site/templates/collection/carousel/item.jet

```diff
@@ -17,7 +17,10 @@
       {{if showItemContent}}
         <div class="carousel-item-caption">
           {{if item.ItemType == "film"}}
-            {{yield sponsor(hideLink="true") item.InnerItem}}
+            <div class="sponsor-awards-wrapper">
```

Comment:
Can you please make this wrapper optional?  If there is non sponsor logo && no award, then there shouldn't be a wrapper either.

Reply:
- Changed to conditional content .jet  wrapper component